pkg/github/github.go

```diff
@@ -478,12 +483,20 @@ func (g *GitHub) downloadAssetsParallel(assets []github.ReleaseAsset, owner, rep
 			if _, err := io.Copy(assetFile, assetBody); err != nil {
 				return errors.Wrap(err, "copying release asset to file")
 			}
-
 			return nil
 		})
 	}
 
-	return errorUtils.AggregateGoroutines(funcs...)
+	for i := 0; i < cap(errChan); i++ {
+		if err := <-errChan; err != nil {
+			if finalErr == nil {
+				finalErr = errors.New(err.Error())
```

Comment:
Any reason we do not want to do:
```suggestion
				finalErr = err
```
Here?

Reply:
Leftover from a previous approach :) done :+1: